code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -619,6 +619,12 @@
 
 /obj/item/nullrod/missionary_staff/proc/do_convert(mob/living/carbon/human/target, mob/living/carbon/human/missionary)
 	var/convert_duration = 6000		//10 min
+
+	// User should be some form of antagonist, otherwise dont let them use the staff
+	if(!missionary.mind.has_antag_datum(/datum/antagonist))
```

Comment:
Ditto for the missionary staff. Let non-antagonist use it too.

Reply:
Done